lib/errors.js

```diff
@@ -115,11 +115,11 @@ Could not parse ${filePath}:
 ${yamlException}
 
 Please make sure that the ui5.yaml has a valid format.
-For reference check: https://github.com/SAP/ui5-project/blob/master/docs/Configuration.md#configuration
+For reference check: https://github.com/SAP/ui5-project/blob/main/docs/Configuration.md#configuration
```

Comment:
We should better link to the UI5 Tooling docs here

Reply:
Done